awesomplete.js

```diff
@@ -281,7 +281,9 @@ function configure(properties, o) {
 			this[i] = attrValue;
 		}
 
-		this[i] = this[i] || o[i] || initial;
+		if (!this[i] && this[i] !== 0) {
+			this[i] = typeof o[i] !== "undefined"? o[i] : initial;
```

Comment:
I'd prefer `!(i in o)` which shorter and more clear. Is there any specific reason you did it this way?